jobs/build/ocp/Jenkinsfile

```diff
@@ -145,6 +145,14 @@ node(TARGET_NODE) {
 
         stage( "openshift-jenkins repo ") {
           buildlib.initialize_openshift_jenkins()
+
+          JENKINS_SOURCE_BRANCH = "master"
+          dir( OPENSHIFT_JENKINS_DIR ) {
+              if(BUILD_VERSION_MAJOR == 3 && BUILD_VERSION_MINOR >= 6 ){
```

Comment:
3.7 is > 3.6, but it is in the master branch right now. You'll need to follow the pattern in 'openshift-ansible prep'.